src/ducks/context/SelectionContext.jsx

```diff
@@ -0,0 +1,70 @@
+import React, {
+  createContext,
+  useState,
+  useCallback,
+  useEffect,
+  useContext
+} from 'react'
+
+import flag from 'cozy-flags'
+
+export const SelectionContext = createContext()
+
+export const useSelectionContext = () => {
+  return useContext(SelectionContext)
+}
+
+const SelectionProvider = ({ children }) => {
+  const [isSelectionModeActive, setIsSelectionModeActive] = useState(false)
+  const [selected, setSelected] = useState([])
+  const isSelectionModeEnabled = flag('banks.selectionMode.enabled')
+
+  const activateSelectionMode = useCallback(
+    () => isSelectionModeEnabled && setIsSelectionModeActive(true),
+    [isSelectionModeEnabled]
+  )
+
+  const deactivateSelectionMode = () => setIsSelectionModeActive(false)
+
+  const addToSelection = item => {
+    setSelected(v => [...v, item])
+  }
```

Comment:
useCallback can be used here

Reply:
même questionnement que pour emptySelection.